tensorflow/core/kernels/BUILD

```diff
@@ -2514,6 +2514,7 @@ tf_kernel_library(
         ":gather_functor",
         ":gpu_prim_hdrs",
         "//tensorflow/core:framework_internal",
+        "//tensorflow/core:gpu_runtime",
```

Comment:
Is this necessary?  `check_numerics_op` does not need this.

Reply:
No, I have removed the header that was requiring that, and forgot to remove this.